CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### New features
 
 * [#117](https://github.com/rubocop/rubocop-minitest/issues/117): Add new cop `Minitest/AssertWithExpectedArgument` to check for unintended usages of `assert` instead of `assert_equal`. ([@cstyles][])
+* [#124](https://github.com/rubocop/rubocop-minitest/pull/124): Add new `Minitest/AnyAssertions` cop. ([@ghiculescu][])
```

Comment:
Can you rebase with the latest mater branch and move this entry to under the `master (unreleased)`?